samples/com.google.android.play/play-services-assetpack/AssetPackSample/MainActivity.cs

```diff
@@ -79,7 +79,7 @@ private void FabOnClick(object sender, EventArgs eventArgs)
 
 		public override void OnRequestPermissionsResult(int requestCode, string[] permissions, [GeneratedEnum] Android.Content.PM.Permission[] grantResults)
 		{
-			Xamarin.Essentials.Platform.OnRequestPermissionsResult(requestCode, permissions, grantResults);
+			Microsoft.Maui.ApplicationModel.Platform.OnRequestPermissionsResult(requestCode, permissions, grantResults);
```

Comment:
Wow, we have to pull in Maui essentials to do this?

Reply:
Well, I "only" migrated Xamarin.Essentials code to MAUI Essentials.

Your samples do build, so I see no big issue. Even better we do test how everything works with MAUI side.